backend/src/pdf_gen/embedded/mod.rs

```diff
@@ -12,7 +12,12 @@ mod world;
 /// Generates a PDF using the embedded typst library.
 pub async fn generate_pdf(file_model: PdfFileModel) -> Result<PdfGenResult, APIError> {
     Ok(tokio::task::spawn_blocking(move || {
-        compile_pdf(&mut world::PdfWorld::new(), file_model.model)
+        let result = compile_pdf(&mut world::PdfWorld::new(), file_model.model);
+
+        // Evict the cache to free up memory + speed up next compile
+        comemo::evict(0);
```

Comment:
Why not make this part of `compile_pdf`?

Reply:
If an error occurs anywhere in `compile_pdf` we still should clear the cache (an thus prevent memory leaks). This way we will always call evict.